typescript/db/src/resolvers/workspace.ts

```diff
@@ -103,6 +110,43 @@ const createWorkspace = async (
     },
     user
   );
+  if (createdWorkspaceId) {
```

Comment:
Code within this block must be factorized into one or multiple functions

Reply:
Fixed